src/org/usfirst/frc4904/robot/Robot.java

```diff
@@ -21,6 +21,7 @@ public void initialize() {
 		operatorChooser.addDefault(new DefaultOperator());
 		// Initialize SmartDashboard display values
 		// SmartDashboard.putNumber(SmartDashboardKey.EXAMPLE.key, 0);
+		RobotMap.Component.navx.zeroYaw();
```

Comment:
@carturn, I couldn't remember where we decided the best place to zero the yaw is - is this right?